src/interface/block_chain.cpp

```diff
@@ -115,7 +118,7 @@ bool block_chain::get_header(chain::header& out_header, size_t height,
 bool block_chain::get_header(chain::header& out_header, size_t& out_height,
     const hash_digest& block_hash, bool block_index) const
 {
-    auto result = database_.blocks().get(block_hash);
+    auto result = database_.blocks().get(block_hash, bitcoin_settings_);
```

Comment:
Don’t pass here, use in database construction.

Reply:
Done.